docs/api/browser-window.md

```diff
@@ -586,6 +586,20 @@ can still bring up the menu bar by pressing the single `Alt` key.
 
 Returns whether the menu bar is visible.
 
+### BrowserWindow.setVisibleOnAllWorkspaces(visible)
+
+* `visible` Boolean
+
+Sets whether the window should be visible on all workspaces.
+
+**Note:** This API is only available on Mac/Linux.
```

Comment:
@zcbenz Done.

@paulcbetts we might support it on Windows 10. But currently this API *does* nothing on Windows. See https://code.google.com/p/chromium/codesearch#chromium/src/ui/views/widget/desktop_aura/desktop_window_tree_host_win.cc&q=SetVisibleOnAllWorkspaces&sq=package:chromium&type=cs&l=360